docs_src/dslx_std.md

```diff
@@ -549,16 +526,16 @@ dumping values to stdout. For example:
 // bazel run -c opt //xls/dslx:interpreter_main  /path/to/dslx/file.x -- --alsologtostderr
```

Comment:
Ugh copybara changes this line to make the doc-test fail due to differing line lengths.

Reply:
I think I'll need to fix this internally.

Reply:
Or actually just moving this note out of the dslx block should be sufficient.

Reply:
Ok I think I did what you were looking for here, but LMK if not!